sematic/api/endpoints/events.py

```diff
@@ -66,6 +66,16 @@ def broadcast_pipeline_update(
     return _call_broadcast_endpoint(url=url, json_payload=json_payload, user=user)
 
 
+def broadcast_job_update(
+    source_run_id: str,
+    user: Optional[User] = None,
+) -> Optional[requests.Response]:
+
+    url = "/events/job/update"
```

Comment:
Note for later: the formation of the `url` should be inside `_call_broadcast_endoing`, which should expect a `namespace` and `action` instead of `url`.